client/my-sites/comments/comment/style.scss

```diff
@@ -492,6 +493,89 @@
 	width: 100%;
 }
 
+// HTML Editor
+
+.comment__html-editor {
+	margin: 0 8px;
+	width: 100%;
+}
+
+.comment__html-toolbar .button {
+	border-right: 1px solid lighten($gray, 30%);
+	border-radius: 0px;
+	color: darken($gray, 20%);
+	margin-bottom: 8px;
+	min-width: 30px;
+	padding: 4px 8px;
+	position: relative;
+	text-transform: lowercase;
+
+	&:hover {
+		color: $gray-dark;
+	}
+
+	&:last-child {
+		border-right: none;
+	}
+
+	&.is-tag-open {
+		padding-left: 12px;
+		padding-right: 4px;
+		&::before {
+			content: '/';
+			left: 8px;
+			position: absolute;
+		}
+	}
+}
+
+.comment__html-toolbar-button-strong {
+	font-weight: bold;
+}
+.comment__html-toolbar-button-em {
+	font-style: italic;
+}
+.comment__html-toolbar-button-a.button {
+	color: $blue-wordpress;
+	text-decoration: underline;
+}
+.comment__html-toolbar-button-del {
+	text-decoration: line-through;
+}
+.comment__html-toolbar-button-ins {
+	text-decoration: underline;
+}
+.comment__html-toolbar-button-code {
+	font-family: $code;
+}
+.comment__html-toolbar-button-close-tags.button[disabled]:hover {
+	color: lighten($gray, 30%);
+}
+
+.editor-html-toolbar__dialog {
```

Comment:
We'll likely delete this later, but is this meant as an override for this section only? The current rule would modify styles for every `editor-html-toolbar__dialog` class

Reply:
This is actually a funnier oversight.
The post editor toolbar's CSS is loaded only with the post editor bundle, so by opening Calypso straight into comments, the dialogs were un-styled.
So I just copied the CSS over to comments, duplicating it without thinking twice.

Good point though: when I'll remove the dialogs I'll also clean up the CSS.
Thanks for pointing it out, I'd have likely missed it. 🙇